mgmtsystem_audit/models/mgmtsystem_audit.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (C) 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import _, api, fields, models
+
+
+class MgmtsystemAudit(models.Model):
+    """Model class that manage audit."""
+
+    _name = "mgmtsystem.audit"
+    _description = "Audit"
+    _inherit = ["mail.thread"]
```

Comment:
nitpicking: I'd take the opportunity to add the activity mixing here, too
What do you think?

Reply:
It was not in mgmtsystem hazard but was in mgmtsystem action.
We can add it because it is audit for audition and it is important.